proxy/main.py

```
@@ -0,0 +1,145 @@
+from constants import INSTRUCTION, MODEL
+
+# Load API key from environment
+# os.environ["GOOGLE_API_KEY"] = ""
+client = genai.Client(
+    http_options={
+        "api_version": "v1alpha",
+    }
+)
+
+
+async def gemini_session_handler(client_websocket: websockets.WebSocketServerProtocol):
```

Pylint (3.3.7)

[refactor] Too many statements (61/50)

(R0915)